bundles/org.eclipse.passage.loc.report.core/src/org/eclipse/passage/loc/report/internal/core/Messages.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2019 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.loc.report.internal.core;
+
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
+
+/**
+ * FIXME
+ * 
+ * @since 0.1
+ */
+public final class Messages {
```

Comment:
Please use NLS style

Reply:
it appears that NLS-style - this only string been localized - brings heavy dependency on `org.eclipse.oeg` bundle. Can we please prevent this from happening? It's is highly unwelcome change for `loc.report.core` - it's intended to be framework-independent. 

This is actually the only string for i10n. I anticipate no mess here.